src/core/EdFi.DataManagementService.Core/ApiSchema/ResourceSchema.cs

```diff
@@ -271,9 +272,9 @@ public DocumentIdentity ExtractDocumentIdentity(JsonNode documentBody)
         if (subclassType == "association")
         {
             return new(
-                ResourceInfo: new(
-                    ResourceName: new(superclassResourceName),
-                    ProjectName: new(superclassProjectName),
+                ResourceInfo: new BaseResourceInfo(
+                    ResourceName: new MetaEdResourceName(superclassResourceName),
```

Comment:
One of the few concrete downsides of using interfaces everywhere: you can't do the target-typed abbreviated `new` anymore.